WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -161,9 +161,9 @@ public async Task<Wallet> StartWalletAsync(Wallet wallet)
 				try
 				{
 					var cancel = CancelAllInitialization.Token;
-					Logger.LogInfo($"Starting {nameof(Wallet)}...");
+					Logger.LogInfo($"Starting Wallet {wallet.WalletName}...");
```

Comment:
How about:

```suggestion
					Logger.LogInfo($"Starting wallet '{wallet.WalletName}'...");
```

Reply:
```
2021-12-25 06:41:28.807 [21] INFO       WalletManager.StartWalletAsync (164)    Starting wallet 'w8'...
```
Looks okay. Made changes in https://github.com/zkSNACKs/WalletWasabi/pull/6940/commits/9258c1cabe87338f001d1b1beffad7729a023bb6